engine/src/main/java/org/hibernate/search/spi/impl/HashSetIndexedTypesSet.java

```diff
@@ -48,6 +49,14 @@ public boolean isEmpty() {
 		return set.stream().map( IndexedTypeIdentifier::getPojoType ).collect( Collectors.toSet() );
 	}
 
+	boolean contains(IndexedTypeIdentifier id) {
+		return set.contains( id );
+	}
+
+	Set<IndexedTypeIdentifier> cloneMap() {
```

Comment:
Yes I mean it's a silly mistake. The whole class was representing Maps, a while back. And cloning them..